apps/web/src/services/asset-inventory/collector/CollectorPage.vue

```diff
@@ -1,5 +1,167 @@
 <template>
     <div class="collector-page">
-        collect-page
+        <p-heading
+            :title="$t('PLUGIN.COLLECTOR.MAIN.TITLE')"
+            use-total-count
+            use-selected-count
+            :total-count="state.totalCount"
+        >
+            <template #extra>
+                <router-link
+                    :to="{ name: ASSET_INVENTORY_ROUTE.COLLECTOR.HISTORY._NAME }"
+                >
+                    <p-button style-type="tertiary"
+                              class="history-button"
+                    >
+                        {{ $t("MANAGEMENT.COLLECTOR_HISTORY.MAIN.TITLE") }}
+                    </p-button>
+                </router-link>
+            </template>
+        </p-heading>
+        <p-data-loader :data="[]"
+                       :loading="state.loading && !state.items"
+        >
+            <!-- TODO: collector contents-->
+            <template #no-data>
+                <collector-no-data />
+            </template>
+        </p-data-loader>
     </div>
 </template>
+
+<script lang="ts" setup>
+import { computed, reactive } from 'vue';
+
+import { PHeading, PButton, PDataLoader } from '@spaceone/design-system';
+
+import { QueryHelper } from '@cloudforet/core-lib/query';
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+import { ApiQueryHelper } from '@cloudforet/core-lib/space-connector/helper';
+
+import { store } from '@/store';
+
+import type { PluginReferenceMap } from '@/store/modules/reference/plugin/type';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+import CollectorNoData from '@/services/asset-inventory/collector/modules/CollectorNoData.vue';
+import type { CollectorModel } from '@/services/asset-inventory/collector/type';
+import { CollectorQueryHelperSet } from '@/services/asset-inventory/collector/type';
+import { ASSET_INVENTORY_ROUTE } from '@/services/asset-inventory/route-config';
+
+const queryHelper = new QueryHelper();
+
+const state = reactive({
+    loading: true,
+    items: undefined as CollectorModel[] | undefined,
+    totalCount: 0,
+    pageStart: 1,
+    pageLimit: 15,
+    sortBy: '',
+    plugins: computed<PluginReferenceMap>(
+        () => store.getters['reference/pluginItems'],
+    ),
+});
+
+/* Query Helper */
+const collectorApiQueryHelper = new ApiQueryHelper()
+    .setOnly(
+        CollectorQueryHelperSet.COLLECTOR_ID,
+        CollectorQueryHelperSet.NAME,
+        CollectorQueryHelperSet.LAST_COLLECTED_AT,
+        CollectorQueryHelperSet.PROVIDER,
+        CollectorQueryHelperSet.TAGS,
+        CollectorQueryHelperSet.PLUGIN_INFO,
+        CollectorQueryHelperSet.STATE,
+    )
+    .setPage(state.pageStart, state.pageLimit)
+    .setSort(state.sortBy, true);
+
+/* API */
+const listCollectors = async () => {
+    state.loading = true;
+    const detailLinkQueryHelper = new QueryHelper();
```

Comment:
suggestion: I recommend to use useQueryTags since the feature related to filter in `QueryHelper` is gonna be deprecated. 😭 
But This is just a suggestion so if there is no enough time, you can skip my comment and do it after implement all the features 😄